rasa/utils/tensorflow/callback.py

```diff
@@ -86,15 +86,20 @@ def _does_model_improve(self, current_results: Dict[Text, Any]) -> bool:
                 self.best_metrics_so_far[key] = float(current_results[key])
             return True
 
-        all_improved = all(
-            [
-                float(current_results[key]) > self.best_metrics_so_far[key]
-                for key in self.best_metrics_so_far.keys()
-            ]
-        )
-
-        if all_improved:
+        at_least_one_improved = False
```

Comment:
Since you've touched this method, could you add a docstring to it as well? (Sorry, I can't attach a comment to a line that you didn't change, hence commenting here.)

Reply:
Yep!